epub33/rs/index.html

```diff
@@ -1172,13 +1172,13 @@ <h4>URL of the Root Directory</h4>
 							with the <a>container root URL</a> as <a data-cite="url#concept-base-url"
 								><var>base</var></a> is the <a>container root URL</a>.</li>
 						<li>The <a href="https://url.spec.whatwg.org/#origin">origin</a> of the <a>container root
-								URL</a> is unique for each EPUB Publication instance.</li>
+								URL</a> is unique for each user-specific instance of an EPUB Publication.</li>
```

Comment:
In #1932 @mattgarrish proposed:
> each user-specific instance of an EPUB Publication **in a Reading System**

The addition of "in a Reading System" implies that the the origin will not be the same for two different RS owned by the same user.

I'm not sure if it was intentionally left out or not. I don't have a definitive opinion on whether it's better with or without.

Reply:
Nop, it was my mistake, simply...